src/ducks/transactions/TransactionRow/TransactionRowMobile.jsx

```diff
@@ -51,7 +52,8 @@ const TransactionRowMobile = ({
   showRecurrence,
   isSelected,
   isSelectionModeActiveFn,
-  toggleSelection
+  toggleSelection,
+  hasSeparator
```

Comment:
I think it would have been a bit better to use the same wording as the component: s/hasSeparator/hasDivider to try not to add too much vocabulary.

Reply:
Effectivement, bien vu 👍